pkg/grpc/universalapi/api_secrets_test.go

```diff
@@ -24,60 +24,14 @@ import (
 	"google.golang.org/grpc/status"
 
 	"github.com/dapr/components-contrib/secretstores"
-	"github.com/dapr/dapr/pkg/apis/resiliency/v1alpha1"
 	"github.com/dapr/dapr/pkg/config"
 	"github.com/dapr/dapr/pkg/messages"
 	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
 	"github.com/dapr/dapr/pkg/resiliency"
 	"github.com/dapr/dapr/pkg/runtime/compstore"
 	daprt "github.com/dapr/dapr/pkg/testing"
-	"github.com/dapr/kit/logger"
-	"github.com/dapr/kit/ptr"
 )
 
-var testLogger = logger.NewLogger("testlogger")
-
-var testResiliency = &v1alpha1.Resiliency{
-	Spec: v1alpha1.ResiliencySpec{
```

Comment:
why is this removed?

Reply:
Was moved to `universalapi_test.go` since it's not only used by secrets: https://github.com/dapr/dapr/pull/6197/files/82d581e4ca6b02857799b4eba8d7f5b60e926aca#diff-f3396dd779d4e21a044dcf5b996f144be2c7911131da8405c5678522356b9070R22